semantic-convention-utils/src/test/java/org/hypertrace/semantic/convention/utils/http/HttpSemanticConventionUtilsTest.java

```diff
@@ -58,6 +58,9 @@
 
 /** Unit test for {@link HttpSemanticConventionUtils} */
 public class HttpSemanticConventionUtilsTest {
+  private static final String HTTP_REQUEST_X_FORWARDED_PROTO =
```

Comment:
If we define `HttpSemanticConventions` as mentioned above, we can use that here.

Reply:
done